components/02-elements/price/Price.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="price">
+    <ins
+      v-if="specialPrice"
+      :aria-label="setAriaLabel(specialPrice)"
+      class="price--special"
+    >
+      {{ specialPrice }}
+    </ins>
+    <del
+      v-if="oldPrice"
+      :aria-label="setAriaLabel(oldPrice)"
+      class="price--old"
+    >
+      {{ oldPrice }}
+    </del>
+    <span
+      v-if="price"
+      :aria-label="setAriaLabel(price)"
+    >
+      {{ price }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    price: {
+      type: String,
+      default: null
+    },
+    specialPrice: {
+      type: String,
+      default: null
+    },
+    oldPrice: {
+      type: String,
+      default: null
+    },
+    ariaLabelPrice: {
+      type: String,
+      default: 'Price:'
+    },
+    ariaLabelSpecial: {
+      type: String,
+      default: 'Special price:'
+    },
+    ariaLabelOld: {
+      type: String,
+      default: 'Old price:'
+    }
+  },
+  methods: {
+    setAriaLabel(type) {
+      if (type === this.specialPrice) {
```

Comment:
you can use `if` `else if` `else`

Reply:
It's better to avoid using `else`, when is possible to end execution via `return`

https://softwareengineering.stackexchange.com/questions/206816/clarification-of-avoid-if-else-advice

Reply:
good point ;)